### ISSUE TITLE
Improve layouting of sequence flows to and from expanded sub-processes

### ISSUE THREAD

**Issue by nikku** [camunda/camunda-modeler#1737]
__Is your feature request related to a problem? Please describe.__

Expanded sub-processes are containers and connections to it (sequence flows) should behave like message flows connecting to participants: They should keep their docking during resize or else resizing the container incurs high risk of breaking existing layouts. 

The current behavior violates our [Do the hard work](https://github.com/bpmn-io/design-principles#do-the-hard-work-to-make-it-simple) design principle, as it is tedious to fix a diagram after bigger resize operations.

![capture xKgPSx_optimized](https://user-images.githubusercontent.com/58601/77306527-1ba97100-6cf8-11ea-8778-a89018d7044c.gif)

Expected behavior in the above screenshot:

* Boundary event positions stay the same, unless the boundary events would not be on the sub-process anymore
* Sequence flows stay intact with their proper layout

Participants behave much better in that regard: 

* Connections (message flows) stick to where they were before the change

__Describe the solution you'd like__

When resizing an expanded sub-process, attachment as well as connection dockings are kept, not magically adjusted. 

__Describe alternatives you've considered__

Maybe the feature is valuable already without keeping keeping attached element positions. 

__Additional context__

See https://github.com/camunda/camunda-modeler/issues/655 for a user request that gets implemented by this.

A solution for shrinking is sketched [in this drawing](https://excalidraw.com/#json=5180254821810176,xHLId6QaD649xaYyDA8l-w).

__TLDR:__ We keep dockings if possible, if not possible we fall back to the current behavior (percentage based).

**Comment by barmac** [comment]
Related to https://github.com/bpmn-io/bpmn-js/issues/1289

**Comment by nikku** [comment]
Closed via https://github.com/camunda/camunda-modeler/commit/79dc1a4e218ac9eabbfc3d1d585821fb12c7a297.
